src/Pheno-Tests-Core.package/PHDummyToolBuilder.class/class/setUp.st

```diff
@@ -0,0 +1,4 @@
+installing
+setUp
+	
+	WidgetsHeap := WeakSet new.
```

Comment:
Hm could we also move all but IsActive to instance side and have the garbagecollector take care of things once the test ran through? No idea if that's the case, but I could imagine this leading to not being able to inspect an interrupted test completely, since we're only holding weak refs

Reply:
Afaik ToolBuilder is instantiated per each object to build? See `UIManager >> toolBuilder`. But if you think it improves the debugging convenience, we could let WidgetsHeap be a normal Set and reset the class variable to nil in `#tearDown`.

Reply:
See 4301d77ab437ed11a4352e785a1d5f0a2fc0cb19.

Reply:
Oh alright, wasn't quite sure about that, thank you :)